### DISCUSSION TITLE
override seo.html

### DISCUSSION THREAD

**Discussion by cedricbompart** [mmistakes/minimal-mistakes#2536] [Q&A] [answered]
I'm trying to override the seo.html because it doesn't do what I want.
When I'm adding a new file in my project under _includes/seo.html; the **bundle exec jekyll serve** is stucked after the _Remote Theme_ output...

Do I need to override something else ?

FYI: I've copied the latest seo.html from the repository.

**Comment by iBug** [comment, marked as answer]
It looks like you're having a network issue connecting to GitHub.

The `jekyll-remote-theme` plugin pulls the latest information from the specified GitHub repository on every build, so you should ensure GitHub reachability when using the remote theme method.

If you're developing locally, the gem-based installation is less of a problem.
